compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEDmaLoopSubsumption.cpp

```diff
@@ -329,6 +357,15 @@ class SubsumeLoopIntoDMA
 
   LogicalResult matchAndRewrite(AMDAIE::DoublyStridedOpInterface op,
                                 PatternRewriter &rewriter) const override {
+    // Get the device model.
+    auto targetAttr = IREE::HAL::ExecutableTargetAttr::lookup(op);
+    std::optional<AMDAIEDevice> device = getConfigAMDAIEDevice(targetAttr);
+    if (!device)
+      return op.emitOpError()
+             << "No AMDAIEDevice found in the target attribute configuration";
+    AMDAIE::AMDAIEDeviceModel deviceModel =
+        AMDAIE::getDeviceModel(AMDAIE::AMDAIEDevice::npu1_4col);
```

Comment:
afaict we're not yet at the point of thinking about supporting different array sizes and architectures, but maybe we should say that explicitly in a meeting (or something)

Reply:
This is a mistake, I meant to do:

```
AMDAIE::AMDAIEDeviceModel deviceModel = AMDAIE::getDeviceModel(device.value());
```